packages/components/src/form-toggle/index.js

```diff
@@ -23,6 +34,12 @@ function FormToggle( { className, checked, id, onChange = noop, ...props } ) {
 			<span className="components-form-toggle__thumb"></span>
 		</span>
 	);
+
+	return disabled ? (
+		<Disabled>{ formToggleInput }</Disabled>
```

Comment:
Instead of wrapping in "Disabled", can't we just add an actual "disabled" property to the "input"?

Reply:
Yes, I just though we would generally prefer `Disabled` component. Just pushed the change with the property only

Reply:
I think we prefer `Disabled` component for complex nested UIs we want to disable but we use just the direct props for single elements.